src/ZipkinBundle/Components/Messenger/ZipkinStamp.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace ZipkinBundle\Components\Messenger;
+
+use Symfony\Component\Messenger\Stamp\StampInterface;
+
+class ZipkinStamp implements StampInterface
```

Comment:
I think this is not specifically a `ZipkinStamp` but a `B3Stamp`. If that is the case, could we move the logic of adding fields inside this one (avoiding the leak of konwledge about propagation in https://github.com/jcchavezs/zipkin-instrumentation-symfony/pull/69/files#diff-cc0918be69c7cb6dc8002c20e67153bbR40-R41). What do you think?

Reply:
Renamed